2021-05-14 12:33:06,230 - sacluster.auth_func_pro.py - ERROR - AuthenticationError: Setting.json does not exist in sacluster/setting
